pkg/collector/corechecks/systemd/systemd.go

```diff
@@ -194,13 +200,15 @@ func (c *SystemdCheck) Run() error {
 	}
 	defer c.stats.CloseConn(conn)
 
+	c.submitVersion(conn)
 	c.submitSystemdState(sender, conn)
 
 	err = c.submitMetrics(sender, conn)
 	if err != nil {
 		return err
 	}
 	sender.Commit()
+
```

Comment:
Unsure if the blank line is on purpose or not.

Reply:
Yeah very minor so let's say so, so that I don't have to run the whole pipeline again (that was quite flaky recently).

We collect the following already: https://docs.datadoghq.com/integrations/systemd/#pagetitle